### PR TITLE
Add user import/export

### PR DESCRIPTION
**Pull request by tjerman**

### CODE REVIEW

File: app/boot_levels.go [hunk 1]
```diff
@@ -507,6 +508,10 @@ func (app *CortezaApp) InitServices(ctx context.Context) (err error) {
 		}
 	}
 
+	// Register reporters
+	// @todo additional datasource providers; generate?
+	sysService.DefaultReport.RegisterReporter("composeRecords", cmpService.DefaultRecord)
```
Comment (darh) [inline app/boot_levels.go:513]: Not sure why is this here?

**Review by darh — COMMENTED** [review]
how hard it would be to add support for uncompressed import?

File: app/boot_levels.go (hunk 1 above)
Comment (tjerman) [reply app/boot_levels.go:513]: I cherry-picked a few bits from the gig branch; this will all get `--ff` to the version branch so I just left it here for the time being
